test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1414,8 +1414,9 @@ def revert_mom(self, mom, force=False):
         self.server.expect(NODE, a, id=mom.shortname, interval=1)
         exp_attrib = {'resources_available.ncpus': (GE, 1)}
         counter = self.server.counter(NODE, exp_attrib)
-        if not counter:
-            self.fail("Value of 'resources_available.ncpus' is less than 1")
+        msg = "Value of 'resources_available.ncpus' is less than 1"
+        rc = counter is not None and counter['resources_available.ncpus>=1'] < 1
+        raise PtlExpectError(rc=rc, msg=msg)
```

Comment:
Two things.  First, if you are going to raise an expect error, why don't you just use expect() and let it handle it?  Second, there is no if here.  If counter['resources_available.ncpus>=1'] < 1, rc will be 0, but you will still raise the expect error.